frontend/src/components/Town/interactables/Music/StepSequencer.tsx

```diff
@@ -80,6 +157,8 @@ export default function MusicArea({
         if (sound.playNote) {
           console.log('Playing Sound: ' + sound.note);
           if (sound.note == 'Cymbol' || sound.note == 'Drum') {
+            // Was synth.triggerAttackRelease('16n', time); but ran into this error in console.log() when spamming
```

Comment:
Can we get this listed as an official bug on the issues page?

Reply:
Done!